packages/gatsby-source-wordpress-next/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "gatsby-source-wordpress",
```

Comment:
Packages in the `packages` directory will get automatically published, so this name should probably match the parent directory name. Otherwise this may get auto-published as a mainline `gatsby-source-wordpress` release.

Reply:
Updated it! Thanks Mike :)